src/partials/components/chart.hbs

```diff
@@ -0,0 +1,13 @@
+<table class="btn btn-{{ type }}" cellpadding="0" cellspacing="0">
```

Comment:
Sorry just noticed this. Is there a reason we're using a btn class here? Assuming it was copied over from another component file?

I think for the purposes of the chart, we can change this to be just one table and give it a different class e.g.

```
<table class="chart" cellpadding="0" cellspacing="0">
  <tr>
   <td align="{{ align }}">
      <img width="{{{ width }}}" src="https://image-charts.com/chart?chs={{{ width }}}x{{{ height }}}&amp;{{{ params }}}" alt="{{{ alt }}}" />
   </td>
  </tr>
</table>

Thoughts?